src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -1550,6 +1551,66 @@ public int GetNextReplicationStatsId()
         {
             return Interlocked.Increment(ref _replicationStatsId);
         }
+
+        internal TestingStuff _forTestingPurposes;
+        internal TestingStuff ForTestingPurposesOnly()
+        {
+            if (_forTestingPurposes != null)
+                return _forTestingPurposes;
+
+            return _forTestingPurposes = new TestingStuff();
+        }
+
+        internal class TestingStuff
```

Comment:
Why do we need all this? can't we use `GetReplicationItems` in both dbs?

Reply:
done